tests/plots/test_station_plot.py

```diff
@@ -451,7 +451,7 @@ def test_stationplot_unit_conversion():
 
 
 def test_scalar_unit_conversion_exception():
-    """Test that errors are raise if unit conversion is requested on un-united data."""
+    """Test that an error is raised if unit conversion is requested on regular arrays."""
```

Comment:
Interchanged usage of `regular` and `plain` here versus above.  Not crucial.

Reply:
Good catch. Fixed.